sdk/communication/AzureCommunicationCommon/Tests/CommunicationIdentifierTest.swift

```diff
@@ -57,25 +57,25 @@ class CommunicationIdentifierTest: XCTestCase {
     }
 
     func test_MicrosoftTeamsUserIdentifier_IfRawIdIsNull_RawIdIsGeneratedProperly_AnonymousUserIsFalse() {
-        let expectedRawId = "8:orgid:User Id"
```

Comment:
I'd keep the hardcoded constants here as they are part of the text fixture - the stable, fixed, well known state.